common/planners/src/plan_builder.rs

```diff
@@ -106,6 +130,29 @@ impl PlanBuilder {
         let input_schema = self.plan.schema();
         let aggr_projection_fields = self.exprs_to_fields(&aggr_expr, &input_schema)?;
 
+        // Aggregator check.
+        for e_aggr in &aggr_expr {
```

Comment:
Good.
But here we need a plan_builder uint test to against the check.
From the code, needs:
1. the passed case
2. the failed case

Reply:
Sure. No problem, happy to work on it.

Reply:
I added three tests in plan_aggregator_test.rs. Do I miss anything?